static/app/utils/analytics/featureFlagAnalyticsEvents.tsx

```diff
@@ -55,4 +60,5 @@ export const featureFlagEventMap: Record<FeatureFlagEventKey, string | null> = {
   'flags.view-all-clicked': 'Clicked View All Flags',
   'flags.view-setup-sidebar': 'Viewed Feature Flag Onboarding Sidebar',
   'flags.cta_read_more_clicked': 'Clicked Read More in Feature Flag CTA',
+  'flags.new_suspect_flags_found': 'New Suspect Flags Found',
```

Comment:
```suggestion
  'flags.suspect_flags_v2_found': 'Suspect Flags V2 Found',
```
🥴 doesn't flow the best but think it's more clear than "new"

Reply:
just renamed